### PR TITLE
Update health-and-diagnostics.md

### PR DESCRIPTION
**Pull request by zryska**
Small set of changes to health and diagnostics section

### CODE REVIEW

File: shared-content/health/health-and-diagnostics.md [hunk 1]
```diff
@@ -6,22 +6,9 @@ uid: HealthAndDiagnostics
...
 
-PI Adapters also produce diagnostic data. You can use diagnostic data to find more information about a particular adapter instance. Diagnostic data lives alongside the health data and you can egress it using a health endpoint and setting `EnableDiagnostics` to`true`. You can configure `EnableDiagnostics` in the system's [General configuration](xref:GeneralConfiguration). For more information on available adapter diagnostics data, see [diagnostics](xref:AdapterDiagnostics).
+PI Adapters also produce diagnostic data. You can use diagnostic data to find more information about a particular adapter instance. Diagnostic data is egressed and stored alongside of the health data. Egress of diagnostics data can be optionally disabled by setting `EnableDiagnostics` to`false`. You can configure it in the system's [General configuration](xref:GeneralConfiguration). For more information on available adapter diagnostics data, see [diagnostics](xref:AdapterDiagnostics).
```
Comment (Xiaoli-OSI) [inline shared-content/health/health-and-diagnostics.md:9]: Rename to AVEVA Adapter? It seems the rename changes have not been merged to 1.7 branch yet.

File: shared-content/health/health-and-diagnostics.md [hunk 2]
```diff
@@ -6,22 +6,9 @@ uid: HealthAndDiagnostics
...
-- OSIsoft Cloud Services (OCS)
-
-There are a few differences in how these two systems treat the associated health and diagnostics data.
-
-- PI Web API parses the information and sends it to configured PI servers for the OMF endpoint. The static data is used to create an AF structure on a PI AF server. The dynamic health data is time-series data that is stored in PI points on a PI Data Archive. You can see it in the AF structure as PI point data reference attributes.
-
-- OCS does not currently provide a way to store the static metadata. For OCS-based adapter health endpoints, only the dynamic data is stored. Each value is its own stream with the timestamp property as the single index.
+In AVEVA Data Hub (ADH), both health and diagnostics data are created as assets. The data are available in the Asset Explorer and you can use them in the ADH Trend feature. For more information, see the ADH documentation [Assets](https://docs.aveva.com/bundle/data-hub/page/add-organize-data/organize-data/assets/asset-concept.html).
 
 ## AF structure
```
Comment (Xiaoli-OSI) [inline shared-content/health/health-and-diagnostics.md:13]: I have some suggestion to reorganize the content for this file, let's talk about it during our sync up meeting.

File: shared-content/health/health-and-diagnostics.md (hunk 1 above)
Comment (zryska) [reply shared-content/health/health-and-diagnostics.md:9]: Rename is part of the `v1.7-Change-Name-AVEVA-Bob` which wasn't merged yet (I initially targeted that branch)

**Review by AnnonsonTechWriter — APPROVED** [review]
Reviewed and approved.

**Pull request merged**